packages/world/src/systems/MineSystem.sol

```diff
@@ -201,7 +233,12 @@ contract MineSystem is System {
       MineNotifData({ mineEntityId: baseEntityId, mineCoord: coord, mineObjectTypeId: mineObjectTypeId })
     );
 
-    ForceFieldLib.requireMinesAllowed(playerEntityId, baseEntityId, mineObjectTypeId, coords, extraData);
+    // TODO: not sure about this
+    if (mineObjectTypeId == ObjectTypes.ForceField && finalMass == 0) {
+      destroyForceField(baseEntityId);
```

Comment:
we still need to call the force field to check if mining is allowed otherwise anyone could mine the force field

Reply:
ideally we should add a test for building and mining a force field